modules/tinymce/src/core/main/ts/lookup/UserLookup.ts

```diff
@@ -261,10 +261,11 @@ const UserLookup = (editor: Editor): UserLookup => {
     }, {});
   };
 
-  const userId = Options.getUserId(editor);
+  const providedUserId = Options.getUserId(editor);
+  const fallbackUserId = 'anonymous';
```

Comment:
Could we make it `"Anonymous"` to align with the current revision history default user?

Reply:
yes

- [ ] rename